src/platforms/java/performance/instrumentation/apollo.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: Apollo Android Integration
```

Comment:
```suggestion
title: Apollo Integration
```

its not Android only right

Reply:
It's not but the package is called "Apollo Android"

Reply:
let's call our package `Apollo Integration` only, otherwise, we could have called the package `sentry-android-apollo` as well, but since it can be used in both, we can be agnostic of the platform